opcua/client.py

```diff
@@ -307,3 +307,6 @@ def get_namespace_array(self):
     def get_namespace_index(self, uri):
         uries = self.get_namespace_array()
         return uries.index(uri)
+
+    def history_read(self, params):
+        return self.bclient.history_read(params)
```

Comment:
maybe method in Client should allow to read several nodes at once.... you decide ....